ndcube/asdf/converters/ndcube_converter.py

```diff
@@ -0,0 +1,30 @@
+from asdf.extension import Converter
+import numpy as np
+
+
+class NDCubeConverter(Converter):
+    tags = ["tag:sunpy.org:ndcube/ndcube/NDCube-*"]
+
+    @property
+    def types(self):
```

Comment:
Strings matching the "class path" can be used here instead of the actual types. Here's an example for the `astropy.column.Column` converter:
https://github.com/astropy/asdf-astropy/blob/1b286108042c1201199e9931cd2a5b52cc57bdce/asdf_astropy/converters/table/table.py#L11
Using a string here has the benefit that `ndcube` does not need to be imported when the `Converter` is "indexed" (which will occur during the first `asdf.open`). There is a small note on this in the [asdf docs](https://asdf.readthedocs.io/en/latest/asdf/extending/converters.html#entry-point-performance-considerations).

Reply:
Right, it would be better to remove it to avoid any unnecessary delay in processing.
Apologies for the delay, I was unwell (Due to fluctuating weather conditions).

Reply:
Thanks for making the update! This is looking great so far. I hope you feel well soon. Please take my frequent comments as excitement about your work :)

Reply:
Thanks for being so informative :)